### PR TITLE
hof 0.6.6 (new formula)

### PR DESCRIPTION
**Pull request by verdverm**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]
LGTM

File: Formula/hof.rb [hunk 1]
```diff
@@ -0,0 +1,46 @@
+class Hof < Formula
+  desc "Flexible data modeling & code generation system"
+  homepage "https://hofstadter.io/"
+  url "https://github.com/hofstadter-io/hof.git",
+    tag:      "v0.6.6",
+    revision: "0f49b4d71a66788b006daacf905a2b138768beca"
```
Comment (carlocab) [inline Formula/hof.rb:6]:
```suggestion
  url "https://github.com/hofstadter-io/hof.git",
      tag:      "v0.6.6",
      revision: "0f49b4d71a66788b006daacf905a2b138768beca"
```
Small nit: align the arguments to `url`.

**Review by carlocab — COMMENTED** [review]
One small suggestion, but otherwise LGTM

File: Formula/hof.rb (hunk 1 above)
Comment (verdverm) [reply Formula/hof.rb:6]: @carlocab Do you want the commits squashed so there is only one ever for this PR?

Comment (carlocab) [reply Formula/hof.rb:6]: Squashing commits would be good. But squashing here is simple, so @BrewTestBot can also handle that. (Sometimes it's not simple, and needs to be done manually.)

**Review by carlocab — DISMISSED** [review]
Thanks!

File: Formula/hof.rb (hunk 1 above)
Comment (verdverm) [reply Formula/hof.rb]: I pushed a second commit, hoping that the bots can handle this case 🤞

**Comment by max-ae** [comment]
This can use the DSL introduced in Homebrew/brew#13536, otherwise the cop from Homebrew/brew#13553 will fail once #105707 and Homebrew/brew#13553 are merged.

File: Formula/hof.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    ENV["CGO_ENABLED"] = "0"
+    ENV["HOF_TELEMETRY_DISABLED"] = "1"
+    system "go", "build", *std_go_args(ldflags: ldflags), "./cmd/hof"
+
+    bash_output = Utils.safe_popen_read(bin/"hof", "completion", "bash")
+    (bash_completion/"hof").write bash_output
+    zsh_output = Utils.safe_popen_read(bin/"hof", "completion", "zsh")
+    (zsh_completion/"_hof").write zsh_output
+    fish_output = Utils.safe_popen_read(bin/"hof", "completion", "fish")
+    (fish_completion/"hof.fish").write fish_output
```
Comment (max-ae) [inline Formula/hof.rb:35]:
```suggestion
    generate_completions_from_executable(bin/"hof", "completion")
```

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**